src/packages/@environments/features/environment-explorer/layout/EnvironmentExplorer.svelte

```diff
@@ -49,12 +59,43 @@
   ) => {
     onUpdateVariable(pairs);
   };
+  let isGuidePopup = false;
+
+  onMount(async () => {
+    const event = await onFetchEnvironmentGuide();
+    console.log(event);
+    if (event.isActive === true) {
+      showContainer = true;
+    } else {
+      showContainer = false;
+    }
+  });
 </script>
 
 {#if $currentEnvironment?.environmentId}
   <div class={`env-panel d-flex`}>
     <div class="env-parent w-100 {quickHelp ? 'quick-help-active' : ''}">
-      <header class={`env-header justify-content-between d-flex`}>
+      <header
+        class={`env-header justify-content-between d-flex`}
+        style="position: relative ;"
+      >
+        {#if $currentEnvironment?.type == "GLOBAL"}
+          <button
+            class="btn p-0"
+            style="position: absolute; left:150px;  top:18px; border:none; z-index:5; curser:pointer;"
+            on:click={() => {
+              showContainer = !showContainer;
+              if (showContainer === true) {
+                onUpdateEnvironmentGuide(true);
+              } else {
+                onUpdateEnvironmentGuide(false);
+              }
+            }}
+          >
+            <HelpIcon height={"12.67px"} width={"12.67px"} />
+          </button>
+        {/if}
```

Comment:
could you please confirm with the design team about this, I think we have to show this for both local and global environment